### PR TITLE
n.makefile

### PR DESCRIPTION
**Pull request by matthew-andrews**
As a separate PR I will switch to using `n-makefile` for builds… which is way faster!

### CODE REVIEW

**Comment by andygnewman** [comment]
LGTM

File: package.json [hunk 1]
```diff
@@ -47,9 +47,13 @@
     "signed-aws-es-fetch": "^1.0.0"
   },
   "devDependencies": {
+    "bower": "^1.7.9",
```
Comment (i-like-robots) [inline package.json]: Could we move this to `optionalDependencies`?

Comment (i-like-robots) [inline package.json]: So local install can use the `--no-optional` flag

Comment (matthew-andrews) [inline package.json]: good idea

**Pull request merged**